library/psa_crypto_cipher.c

```diff
@@ -304,7 +322,6 @@ static psa_status_t psa_cipher_update_ecb(
                 goto exit;
 
             output += internal_output_length;
```

Comment:
Wouldn't it be better to check here if we have enough room in the output buffer?

Reply:
I chose not to add the check because this function has a single caller, in the same module, and the check would be exactly the same check that the caller makes (and which applies to other cases so we can't just move that check, we'd have to copy it).